review.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+#
+# AWS Lambda Code Grabber
+# v1.0 - TJ
+#
+# Grab all Lambda code bases in one mighty swoop
+# Yes yes, it is just a wrapper around aws :)
+#
+import os
+import sys
+import subprocess
+import errno
+import json
+import argparse
+
+# Requirements
+AWS="/usr/local/bin/aws"
+JQ="/usr/bin/jq"
+
+# Define the directory that's going to store all this
+FUNDIR="functions"
+
+
+
+##### Get awf my land  ######
+
+def banner():
+    print("""
+           __  _
+       .-.'  `; `-._  __  _
+      (_,         .-:'  `; `-._
+    ,'o"(        (_,           )
+   (__,-'      ,'o"(            )>
+      (       (__,-'            )
+       `-'._.--._(             )
+          |||  |||`-'._.--._.-'
+                     |||  |||
+
+             \033[0;32mLambshank v1.0 
```

Comment:
If you're doing lots of colour in the terminal, you could give the ascii escape code variables

Reply:
ok 👍